2022/34xxx/CVE-2022-34872.json

```diff
@@ -1,18 +1,70 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2022-34872",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "CVE_data_meta": {
+    "ASSIGNER": "zdi-disclosures@trendmicro.com",
+    "ID": "CVE-2022-34872",
+    "STATE": "PUBLIC"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Centreon",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_value": "21.10-2"
+                    }
+                  ]
+                }
+              }
+            ]
+          },
+          "vendor_name": "Centreon"
+        }
+      ]
+    }
+  },
+  "credit": "Anonymous",
+  "data_format": "MITRE",
+  "data_type": "CVE",
+  "data_version": "4.0",
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "This vulnerability allows remote attackers to disclose sensitive information on affected installations of Centreon. Authentication is required to exploit this vulnerability.\n\nThe specific flaw exists within the processing of Virtual Metrics. The issue results from the lack of proper validation of a user-supplied string before using it to construct SQL queries. An attacker can leverage this vulnerability to disclose stored credentials, leading to further compromise. Was ZDI-CAN-16336."
+      }
+    ]
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection')"
+          }
         ]
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "url": "https://www.zerodayinitiative.com/advisories/ZDI-22-954/"
+      },
+      {
+        "url": "https://docs.centreon.com/docs/21.10/releases/centreon-core/\r\n\r\nhttps://github.com/centreon/centreon/blob/master/SECURITY_ACK.md#-2022-"
```

Comment:
Looks like a bug in automation here. Two urls listed in this string.

Reply:
The vendor sends us that and a human puts it in. What baffles me is, why did my JSON schema validate, but yours caught it. I bet our schema isn't caught up to the latest. 

Thanks. Updated.